### DISCUSSION TITLE
How can I specify the window size?

### DISCUSSION THREAD

**Discussion by rovf** [wezterm/wezterm#5932] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

How can I specify the numbers of rows and columns for the terminal being created?

Neither https://wezfurlong.org/wezterm/config/files.html#configuration-file-structure not the FAQ show this.

**Comment by wez** [comment, marked as answer]
https://wezfurlong.org/wezterm/config/lua/config/initial_cols.html
https://wezfurlong.org/wezterm/config/lua/config/initial_rows.html
